src/main/java/budgetbuddy/model/loan/Status.java

```diff
@@ -24,6 +26,16 @@
         this.status = status;
     }
 
+    /**
+     * Returns true if a given string corresponds to a Status value.
+     * @param toTest The string to test.
+     */
+    public static boolean contains(String toTest) {
```

Comment:
Maybe might want to consider renaming this method. `Direction.contains("some string");` seems a little off. May I suggest `Direction.of`?

Reply:
Hm but the main purpose of the method is to check if the enum contains a value matching the string, which is why it returns a boolean. `Direction.of` sounds quite similar to the existing `Direction.valueOf` , which returns a `Direction` or throws an exception if the value is not found.